tx/textual/internal/testdata/any.json

```diff
@@ -51,8 +51,8 @@
             }
         },
         "screens": [
-            {"text": "Object: /google.protobuf.Any"},
-            {"text": "Object: /Foo", "indent": 1},
+            {"text": "/google.protobuf.Any"},
+            {"text": "/Foo", "indent": 1},
```

Comment:
The only weird output would be this.

The first line will have, in a message context, a field name:
```
<Field>: /google.protobuf.Any
```

But the 2nd line will be a naked `/Foo`. I think it's fine, because nested packed Anys should be extremely rare.